packages/framework/src/Form/Constraints/MoneyRange.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Form\Constraints;
+
+use Shopsys\FrameworkBundle\Component\Money\Money;
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ */
+class MoneyRange extends Constraint
+{
+    /**
+     * @var string
+     */
+    public $minMessage = 'The amount of money should be {{ limit }} or more.';
+
+    /**
+     * @var string
+     */
+    public $maxMessage = 'The amount of money should be {{ limit }} or less.';
+
+    /**
+     * @var \Shopsys\FrameworkBundle\Component\Money\Money|null
+     */
+    public $min;
+
+    /**
+     * @var \Shopsys\FrameworkBundle\Component\Money\Money|null
+     */
+    public $max;
+
+    /**
+     * @param array|null $options
+     */
+    public function __construct($options = null)
```

Comment:
```suggestion
    public function __construct($options)
```
There is no way `$options` can be nullable and this class instantiated because at least min or max must be given

Reply:
Makes sense, `$options` has to be `array`.